conversion/conversion_from_source.go

```diff
@@ -59,18 +59,32 @@ func (sads *SchemaFromSourceImpl) schemaFromDatabase(migrationProjectId string,
 	var infoSchema common.InfoSchema
 	var err error
 	isSharded := false
+	fmt.Printf("fetching schema from database %+v\n", sourceProfile.Config.ShardConfigurationBulk)
```

Comment:
This might print sensitive details to the log? Please check print statements added (in other parts of PR as well)